docs: update relayer KB to add a remove existing relayer section (#2022)
tendermint/starport: docs/kb/relayer.md

By default, relayer configuration is stored in `$HOME/.relayer/`.

## Remove existing relayers

If you previously used the Starport relayer, follow these steps to remove existing relayer and Starport configurations:

1. Stop your blockchain or blockchains.

> 2. Delete previous configuration files:
> 
>     ```bash
>     rm -rf ~/.starport/relayer
>     ```
> 
>     If existing configurations do not exist, the command returns `no matches found` and no action is taken.

Reviewer: I think if you use the `-f` flag the return will be always empty in most shells. 

```suggestion
2. If a previous configuration exist, you should delete the config files:

    ```bash
    rm -rf ~/.starport/relayer
    ```
```
